src/controllers/gameController.js

```diff
@@ -7,26 +7,31 @@
     this.intervalTimer;
     this.bindKeys();
     this.updateGame(this.game.car);
+    this.keys = {};
   }
 
   GameController.prototype.bindKeys = function () {
-    window.addEventListener('keyup', this._pressKeyHandler, false);
+    window.addEventListener('keyup', this._keyupHandler, false);
+    window.addEventListener('keydown', this._keydownHandler, false);
   };
 
   GameController.prototype.unbindKeys = function () {
-    window.removeEventListener('keyup', this._pressKeyHandler, false);
+    window.removeEventListener('keyup', this._keyupHandler, false);
+    window.removeEventListener('keydown', this._keydownHandler, false);
   };
 
-  GameController.prototype.spaceBarKeyPressed = function (key) {
-    if(key.keyCode === 32){
-      if(this.countdownFinished) {
-        this.game.car.accelerate();
-      }
+  GameController.prototype.keyup = function (key) {
+    if(key.keyCode == 32){
+       if(this.countdownFinished) {
+         this.game.car.accelerate();
+       }
     }
+      this._removeKey(key);
   };
 
-  GameController.prototype.enterKeyPressed = function (key) {
-    if(key.keyCode === 13){
+  GameController.prototype.keydown = function (key) {
```

Comment:
We should move the checking of enter/start countdown to a more generic method that handles all key down events